src/Filament/Resources/ExceptionLogResource.php

```diff
@@ -15,7 +15,7 @@
 
 class ExceptionLogResource extends Resource
 {
-    protected static ?string $slug = 'moonguard/exceptions';
+    protected static ?string $slug = 'exceptions';
```

Comment:
Is this change intended? seems like you deleted the `moonguard/`.

Reply:
yes, because in v3 you can define a general prefix in the service provider.